### ISSUE TITLE
Common name problems

### ISSUE THREAD

**Issue by ctc3d** [EDIorg/taxonomyCleanr#30]
Docs error?
The Common Use Case says:
	**Resolve common taxa**
	`# View the list of authorities supported by resolve_comm_taxa`
	`view_authorities()`
I get
	# Error in view_authorities() : could not find function "view_authorities"
`view_taxa_authorities()` gives the information for common, so that's okay, but the instructions are off.

`resolve_comm_taxa()` itself isn't very useful, seeming to just add the common name to `taxa_clean`.  I assume that's a limitation of the authorities, but it's injecting a different kind of data into that field; I didn't find any cases where information was added or improved.

**Comment by clnsmth** [comment]
Thanks @ctc3d. I'll fix the documentation issue ASAP.

Adding the common name to `taxa_clean` is the expected behavior (i.e. if the name is not of the scientific nomenclature then is it colloquial?). Perhaps a separate field for common names would be more useful so both types of information can be included?

**Comment by ctc3d** [comment]
Yeah, a separate field like `common_clean` or something might be a good idea.  My immediate problem is that some of these inventories -- curse them -- *only* have common names, which are hard to do much with at our next step.  I was hoping TCR/ITIS might take a stab at guessing a scientific name from a common name -- but I recognize that that's probably a little Holy Grailish.

I can see how for any results more advanced than what TCR is already providing it kind of has to work interactively, with the whole other ball of wax that that implies and so could be beyond your reasonable scope.
